src/Storage/Storage.Management.Sdk/README.md

```diff
@@ -1,3 +1,14 @@
+# Overall
+This directory contains management plane service clients of Az.Storage module.
+
+## Run Generation
+In this directory, run AutoRest:
+```
+autorest.cmd README.md --version=v2
```

Comment:
Maybe we should add following command before this line to make sure using the correct version of autorest:

```
autorest --reset
autorest --use:@microsoft.azure/autorest.csharp@2.3.90
```